Misc/NEWS.d/next/Library/2018-03-25-13-18-16.bpo-33096.ofdbe7.rst

```diff
@@ -1,4 +1,3 @@
 Allow ttk.Treeview.insert to insert iid that has a false boolean value.
 Note iid=0 and iid=False would be same.
 Patch by Garvit Khatri.
-
```

Comment:
Editing error which blurb should fix or report as 'trailing blank'.

Reply:
I spent an hour or two at the CPython core dev sprints trying to figure out how blurb could have possibly made this mistake.  blurb is super-aggressive about reformatting your text, and even *more* aggressive about trailing whitespace.

@zooba figured out what happened: the author used "blurb" to originally create this entry, but didn't finish the last sentence with a period.  @serhiy-storchaka asked the author to fix it.  When the author added the trailing period, *by editing the file manually in a text editor,* they also introduced the trailing newline.

https://github.com/python/cpython/pull/6228/commits/2e2defa268c2272614f66f77886b6b70106be62f

I guess it's too much to expect people to stop blaming blurb for other people's mistakes.  But perhaps in the future you will keep this in mind, and not be so quick to blame blurb.